### PR TITLE
Updating CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by nashitshayan**
Made additions and updated the doc - making the setup steps more clearer and reducing assumptions made on part of the reader.

### CODE REVIEW

**Review by shubhamsigdar1 — COMMENTED** [review]
Hey @nashitshayan  I want to ask How can we Generate a set of RSA keys 2048 bit in size??

**Comment by nashitshayan** [comment]
@shubhamsigdar1 No idea. Will look it up tonight and update the doc accordingly. 
I suspect that this is one of the keys that we get from the Discord Bot dashboard, because Ritik only added this later, there was no mention of RSA previously and I had got my test bot working just fine by following the old doc. 
This is the kind of confusion that we need to remove from our docs.

**Review by prakashchoudhary07 — DISMISSED** [review]

**Comment by prakashchoudhary07** [comment]
Once get it reviewed by @RitikJaiswal75

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -43,24 +51,23 @@ Please set Following permissions for your bot:
...
-- Clone the Repository to your machine
-- Now, get the .env file created above in the project folder
 - run `npm install`
-- Now, run the command `npm run register`
-
-This will register all the commands to your discord bot.
+- Now, run the command `npm run register` - this will register all the commands to your discord bot.
 
-Now let's link our local development server to our bot.
+Next you will have to set up the wranler cli, so that you can connect to your cloudflare workers account.
```
Comment (samyakshah3008) [inline CONTRIBUTING.md:65]: Nitpik: it should be `wrangler cli` instead of `wranler cli` right?

**Review by samyakshah3008 — APPROVED** [review]
Great work! Thanks for improving this documentation! @nashitshayan 👌

File: CONTRIBUTING.md (hunk 1 above)
Comment (nashitshayan) [reply CONTRIBUTING.md:65]: Hehe nice catch!

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -5,29 +5,37 @@
...
-Copy the guild id of your server and save it in a .env file as `DISCORD_GUILD_ID`
-
-To get the server ID for the first parameter, open Discord, go to Settings > Advanced and enable developer mode.
-Then, right-click on the server title and select "Copy ID" to get the guild ID.
-
+To contribute to this project, you have to set up few things first.
+
+- Create an account on Cloudfare Workers and Discord if you don't already have one
+- Create a personal discord server
+- Fork and clone this repo
```
Comment (RitikJaiswal75) [inline CONTRIBUTING.md:12]: this is optional, If you are a member of the community you don't have to do this.

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -5,29 +5,37 @@
...
+- Install volta - this will check if you have the right node version or not
+- Create a .env file
```
Comment (RitikJaiswal75) [inline CONTRIBUTING.md:14]: mention that to be created in root directory

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -5,29 +5,37 @@
...
 
-Gather the following details from the Developer portal and save it in a .env file.
+Next, you need to create your own bot. The idea here is that you will have your own bot that can be used to run all commands in your private server. You can add new features and test it there to make sure everything works as expected, before deploying the code on production.
 
-```
-DISCORD_TOKEN: Available in bot panel of your discord bot after clicking reset token button.
-DISCORD_APPLICATION_ID: Available in general panel of your discord bot.
-DISCORD_PUBLIC_KEY: Available in general panel of your discord bot.
-```
+- To create a bot, go to the Bot panel and click on 'Add Bot'
```
Comment (RitikJaiswal75) [inline CONTRIBUTING.md:32]: Isn't it the direct applications panel?

File: CONTRIBUTING.md (hunk 2 above)
Comment (nashitshayan) [reply CONTRIBUTING.md:12]: you still have to clone right?

File: CONTRIBUTING.md (hunk 3 above)
Comment (nashitshayan) [reply CONTRIBUTING.md:14]: done

File: CONTRIBUTING.md (hunk 4 above)
Comment (nashitshayan) [reply CONTRIBUTING.md:32]: add_bot_discord

File: CONTRIBUTING.md (hunk 2 above)
Comment (RitikJaiswal75) [reply CONTRIBUTING.md]: yes

File: CONTRIBUTING.md (hunk 4 above)
Comment (RitikJaiswal75) [reply CONTRIBUTING.md:32]: you will find it in oauth 2 URL generator

Comment (RitikJaiswal75) [reply CONTRIBUTING.md:32]: is it directly available when you have no bots? i don't think this interface opens up without any bot created

Comment (nashitshayan) [reply CONTRIBUTING.md:32]: @RitikJaiswal75  in any case, the 'bot' option is always available in the side panel

**Review by RitikJaiswal75 — APPROVED** [review]
LGTM

**Pull request merged**